### PR TITLE
Rs 2022.12.01

### PR DESCRIPTION
**Pull request by rschlaikjer**
「ま゛？」
2022-12-01-223109_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 10/10_01_CIEL10_4_1.txt [hunk 1]
```diff
@@ -126,57 +141,62 @@ Or even if she drinks herself silly after work."
...
 -- Page 181, Offset 27553.
 -- ノエル先生、教師としては若いといっても25歳だし」
-Noel-sensei, though young for a teacher, is twenty-five years old."
+Noel-sensei's a bit young for a teacher, but she's still twenty-five."
 }
 [sha:283ee2efbd144ad7cf0517dc3a1fc2287fcb3c37]{
 -- Page 181, Offset 27554.
 -- 「ま゛？」
-"Wha...?"
+"Wha...?!"
```
Comment (Petrikow) [inline script/Ciel/Day 10/10_01_CIEL10_4_1.txt:188]: I think em-dash is more appropriate.

Though to be fair I don't think we've quite established a standard on interruptions.

File: script/Ciel/Day 10/10_01_CIEL10_4_1.txt [hunk 2]
```diff
@@ -126,57 +141,62 @@ Or even if she drinks herself silly after work."
...
 -- 再度、ものすごい声がした。
-Once again, that was an astonishing reaction.
+That makes two dramatic reactions.
 }
 [sha:8796bdf9a69358d98987b9e3c11d27fe1196f0b2]{
 -- Page 182, Offset 27556.
 -- どんな発言も笑顔で受け止める先輩とは思えない、驚愕の威嚇音だった。
-A frightening sound of astonishment that I would never have expected to hear from Senpai, who normally takes in any kind of proclamation with a smile on her face.
+I'm used to seeing Senpai take basically anything I say in stride,
+ but my comment elicited what sounds like almost hostile surprise.
```
Comment (Petrikow) [inline script/Ciel/Day 10/10_01_CIEL10_4_1.txt:199]: iirc 威嚇音 is like a warning sound. Somehow I feel like this comparison could be smoothed out a bit. Not entirely sure what's appropriate.

>...but it seems my comment has left her almost aggressively surprised.

Yeah, it's a weird one. I dunno

File: script/Ciel/Day 10/10_01_CIEL10_4_1.txt [hunk 3]
```diff
@@ -186,16 +206,20 @@ A frightening sound of astonishment that I would never have expected to hear fro
 [sha:5ea6c7d9e2480693d82724dfbd36b42f02be4951]{
 -- Page 183, Offset 27558.
 -- 「……25歳って……潜入任務ですから、多少の偽装はするでしょうけど……シスター・ノエル……」
-"Twenty-five years old, she says... An undercover operation needs some degree of disguise, but... Sister Noel..."
+"Twenty-five years old, she says...
+ I appreciate that an undercover mission calls for a disguise, but... Sister Noel..."
```
Comment (Petrikow) [inline script/Ciel/Day 10/10_01_CIEL10_4_1.txt:210]: I think the tone should be a bit more.. mocking?
>Really, twenty-five years old...
>I mean, given, undercover missions do call for some level of disguise, but... Sister Noel...

File: script/Ciel/Day 10/10_01_CIEL10_4_1.txt [hunk 4]
```diff
@@ -186,16 +206,20 @@ A frightening sound of astonishment that I would never have expected to hear fro
...
 }
 [sha:9dd05b658a3f1292044f0132336abe3f748a9830]{
 -- Page 184, Offset 27559.
 -- ヤバイ。この話題、これ以上踏み込むと危険な気がする。後でノエル先生に背後から襲われかねない。
 // Noel-chan is a sweet girl, she'd never do that!
-Crap. I fear that continuing on that topic any longer will be dangerous. Noel-sensei could very well assault me from behind later on.
+Crap.
+ I feel like digging any deeper on this subject is going to land me in trouble.
+ Noel-sensei might ambush me and break my kneecaps later.
```
Comment (Petrikow) [inline script/Ciel/Day 10/10_01_CIEL10_4_1.txt:218]: Break my kneecaps...?

File: script/Ciel/Day 10/10_01_CIEL10_4_1.txt [hunk 5]
```diff
@@ -36,87 +39,99 @@ A full-strength denial, uttered in a tone wholly unsuited to how calm she usuall
...
+"A-Ah, well... It's true that Sister Noel has been my partner for many years.
 }
 [sha:506c99835d5c7a441ee43f129e0d3565f8d3e4d7]{
 -- Page 173, Offset 27531.
 -- あの……遠野くんにはそう見えているんですか？　あと、彼女がわたしを褒めていたんですか？　本当に？」
-Um... Does it look like this to you, Tohno-kun? And... She praised me? Really?"
+// I would have expected this 'Uhh is that really how it looks to you, buddy' to be more sarcastic, but the VA sounds kinda earnest?
+// Next line also kinda makes it sound like she's actually worried about it
+Um... is that really your honest impression, Tohno-kun?
+ And... you said she praised me? Is that true?"
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 10/10_01_CIEL10_4_1.txt:61]: Hmm, not sure I agree. In expressions like `have nothing but praise for` yes, but here it sounds like a slightly archaic phrasing to my ear

File: script/Ciel/Day 10/10_01_CIEL10_4_1.txt (hunk 3 above)
Comment (rschlaikjer) [reply script/Ciel/Day 10/10_01_CIEL10_4_1.txt:210]: VA kinda sounds more exasperated to me here than mocking

Comment (Petrikow) [reply script/Ciel/Day 10/10_01_CIEL10_4_1.txt:210]: I mean, that's why it's just a bit more.

**Pull request merged**